src/entitysdk/models/agent.py

```diff
@@ -1,18 +1,27 @@
 """Agent models."""
 
-from entitysdk.models.core import Struct
+from typing import Literal
 
+from entitysdk.models.core import Identifiable
 
-class Person(Struct):
+
+class Agent(Identifiable):
+    """Agent model."""
+
+    type: str
+    pref_label: str
+
+
+class Person(Agent):
     """Person model."""
 
-    givenName: str
+    type: Literal["person"]
```

Comment:
I would be in favor to make type a universal property by adding it to either BaseModel or Identifiable.

Reply:
I think it makes sense, we can add it after changing entitycore